lib/lotus/web/components/layouts.ex

```diff
@@ -29,7 +29,7 @@ defmodule Lotus.Web.Layouts do
 
   def logo(assigns) do
     ~H"""
-    <a href={lotus_path("")} title="Lotus Web">
+    <a href={lotus_path("")} title={gettext("Lotus Web")}>
```

Comment:
This is the brand and there’s no plan to offer Lotus as a white label, so I dont see a good reason to translate this string.